gradle/project.gradle.kts

```diff
@@ -176,12 +172,12 @@ tasks {
         onlyIf { !dirJdk.exists() }
         doFirst {
             println("Making JDK locally accessible...")
-            val jdkPath = "java.home".sysProp.takeIf { it.isNotBlank() }?.let { Paths.get(it) } ?: failIO { "Unable to find JDK" }
+            val jdkPath = "java.home".sysProp?.let { Paths.get(it) } ?: failIO { "Unable to find JDK" }
             try {
                 Files.createSymbolicLink(dirJdk.toPath(), jdkPath)
             } catch (e: Exception) {
                 println("Couldn't create a symbolic link from $dirJdk to $jdkPath: $e")
-                if ("os.name".sysProp.startsWith("Windows")) {
+                if ("os.name".sysProp?.startsWith("Windows")==true) {
```

Comment:
Correct.

Reply:
Do you mean that it is correct or that I should correct it?

Reply:
it is correct.